feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -79,7 +79,7 @@ fun ProfileView(
         Row(modifier = Modifier.fillMaxWidth()) {
             Column(modifier = Modifier.padding(horizontal = sizes.smallSize)) {
                 Image(
-                    painter = painterResource(id = R.drawable.default_user_photo),
+                    painter = painterResource(id = com.iteneum.designsystem.R.drawable.default_user_photo),
```

Comment:
To avoid the `com.iteneum.designsystem` and add it to the import side you can use an alias `as` to name the import example: 
`import com.iteneum.designsystem.R as R_DS`

Reply:
Done